mmv1/products/bigquery/api.yaml

```diff
@@ -130,6 +130,7 @@ objects:
               must contain only letters (a-z, A-Z), numbers (0-9), or
               underscores (_). The maximum length is 1,024 characters.
             required: true
+            input: true
```

Comment:
The same is true of `projectId` and the parent `datasetReference`, right?

Reply:
yes, that makes sense!